### ISSUE TITLE
Itemsadder WorldEdit not working with FAWE (1.19.2)

### ISSUE THREAD

**Issue by BigTows** [PluginBugs/Issues-ItemsAdder#2457]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

BigTows#4856

### What happened?

 Itemsadder WorldEdit not compaliable with 1.19.2 (

### Steps to reproduce the issue

1. Setup 
   * [Paper (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: 476ef25)](https://api.papermc.io/v2/projects/paper/versions/1.19.2/builds/307/downloads/paper-1.19.2-307.jar)
   *  [FastAsyncWorldEdit-Bukkit-2.6.0-SNAPSHOT-392.jar](https://ci.athion.net/job/FastAsyncWorldEdit/392/)
   * [ItemsAdder_3.4.1-r2](https://www.spigotmc.org/resources/%E2%9C%A8itemsadder%E2%AD%90emotes-mobs-items-armors-hud-gui-emojis-blocks-wings-hats-liquids.73355/update?update=494094)
   * [ProtocolLib 606](https://ci.dmulloy2.net/job/ProtocolLib/606/)
   * [Itemsadder WorldEdit 1.1.2-r2](https://www.spigotmc.org/resources/addon-itemsadder-worldedit.79012/update?update=492622)
2. Launch 
3. Place custom block from ItemAdder
4. Select block with a wand axe.
5. Perform command //copy, //paste
6. Click on the pasted block
7. yeee note block :) 
![image](https://user-images.githubusercontent.com/9078504/227765108-6a31f89a-ee4e-4264-8c97-2330d7ce67c3.png)

### Server version

This server is running Paper version git-Paper-307 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: 476ef25)
You are running the latest version
Previous version: git-Paper-448 (MC: 1.19.3)

### ItemsAdder Version

ItemsAdder version 3.4.1-r2

### ProtocolLib Version

 ProtocolLib version 5.0.0-SNAPSHOT-b606

### LoneLibs Version

LoneLibs version 1.0.24

### FULL server log

```
*** Warning, you've not updated in a while! ***
*** Please download a new build as per instructions from https://papermc.io/downloads ***
System Info: Java 19 (OpenJDK 64-Bit Server VM 19+36-2238) Host: Linux 5.15.90.1-microsoft-standard-WSL2 (amd64)
Loading libraries, please wait...
[12:00:28 INFO]: Building unoptimized datafixer
[12:00:30 INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[12:00:34 INFO]: Loaded 7 recipes
[12:00:34 INFO]: Starting minecraft server version 1.19.2
[12:00:34 WARN]: ****************************
[12:00:34 WARN]: YOU ARE RUNNING THIS SERVER AS AN ADMINISTRATIVE OR ROOT USER. THIS IS NOT ADVISED.
[12:00:34 WARN]: YOU ARE OPENING YOURSELF UP TO POTENTIAL RISKS WHEN DOING THIS.
[12:00:34 WARN]: FOR MORE INFORMATION, SEE https://madelinemiller.dev/blog/root-minecraft-server/
[12:00:34 WARN]: ****************************
[12:00:34 INFO]: Loading properties
[12:00:34 INFO]: This server is running Paper version git-Paper-307 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: 476ef25)
[12:00:35 INFO]: Server Ping Player Sample Count: 12
[12:00:35 INFO]: Using 4 threads for Netty based IO
[12:00:35 INFO]: [ChunkTaskScheduler] Chunk system is using 1 I/O threads, 3 worker threads, and gen parallelism of 3 threads
[12:00:35 INFO]: Default game type: SURVIVAL
[12:00:35 INFO]: Generating keypair
[12:00:35 INFO]: Starting Minecraft server on *:25565
[12:00:35 INFO]: Using epoll channel type
[12:00:35 INFO]: Paper: Using libdeflate (Linux x86_64) compression from Velocity.
[12:00:35 INFO]: Paper: Using OpenSSL 3.0.x (Linux x86_64) cipher from Velocity.
[12:00:36 INFO]: [FastAsyncWorldEdit] Loading FastAsyncWorldEdit v2.6.0-SNAPSHOT-392;d82bf05
[12:00:37 INFO]: Got request to register class com.sk89q.worldedit.bukkit.BukkitServerInterface with WorldEdit [com.sk89q.worldedit.extension.platform.PlatformManager@2cf58010]
[12:00:37 INFO]: [LoneLibs] Loading LoneLibs v1.0.24
[12:00:37 INFO]: [ProtocolLib] Loading ProtocolLib v5.0.0-SNAPSHOT-b606
[12:00:37 WARN]: [ProtocolLib] Version (MC: 1.19.2) has not yet been tested! Proceed with caution.
[12:00:38 INFO]: [ItemsAdder] Loading ItemsAdder v3.4.1-r2
[12:00:38 INFO]: [ItemsAdder-WorldEdit] Loading ItemsAdder-WorldEdit v1.1.2-r2
[12:00:38 INFO]: Server permissions file permissions.yml is empty, ignoring it
[12:00:38 INFO]: [FastAsyncWorldEdit] Enabling FastAsyncWorldEdit v2.6.0-SNAPSHOT-392;d82bf05
[12:00:39 INFO]: [com.fastasyncworldedit.core.Fawe] LZ4 Compression Binding loaded successfully
[12:00:39 INFO]: [com.fastasyncworldedit.core.Fawe] ZSTD Compression Binding loaded successfully
[12:00:39 INFO]: Registering commands with com.sk89q.worldedit.bukkit.BukkitServerInterface
[12:00:39 INFO]: WEPIF: Using the Bukkit Permissions API.
[12:00:39 INFO]: Using com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_19_R1.PaperweightFaweAdapter as the Bukkit adapter
[12:00:40 INFO]: [LoneLibs] Enabling LoneLibs v1.0.24
[12:00:40 INFO]: [ProtocolLib] Enabling ProtocolLib v5.0.0-SNAPSHOT-b606
[12:00:40 WARN]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[12:00:40 WARN]: The server will make no attempt to authenticate usernames. Beware.
[12:00:40 WARN]: While this makes the game possible to play without internet access, it also opens up the ability for hackers to connect with any username they choose.
[12:00:40 WARN]: To change this, set "online-mode" to "true" in the server.properties file.
[12:00:40 INFO]: Preparing level "world"
[12:00:41 INFO]: Preparing start region for dimension minecraft:overworld
[12:00:41 INFO]: Time elapsed: 153 ms
[12:00:41 INFO]: Preparing start region for dimension minecraft:the_nether
[12:00:41 INFO]: Time elapsed: 22 ms
[12:00:41 INFO]: Preparing start region for dimension minecraft:the_end
[12:00:41 INFO]: Time elapsed: 20 ms
[12:00:41 INFO]: [ItemsAdder] Enabling ItemsAdder v3.4.1-r2
[12:00:42 INFO]: [NBTAPI] [NBTAPI] Found Spigot: v1_19_R1! Trying to find NMS support
[12:00:42 INFO]: [NBTAPI] [NBTAPI] NMS support 'MC1_19_R1' loaded!
[12:00:42 INFO]: [NBTAPI] [NBTAPI] Found Gson: class com.google.gson.Gson
[12:00:42 INFO]: [ItemsAdder]
  ___  ___        __        __   __   ___  __      ItemsAdder 3.4.1-r2
|  |  |__   |\/| /__`  /\  |  \ |  \ |__  |__)     LoneLibs 1.0.24
|  |  |___  |  | .__/ /--\ |__/ |__/ |___ |  \     Paper git-Paper-307 (MC: 1.19.2)

[12:00:43 INFO]: [ItemsAdder-WorldEdit] Enabling ItemsAdder-WorldEdit v1.1.2-r2
[12:00:43 INFO]: Running delayed init tasks
[12:00:43 INFO]: [ItemsAdder] [Pack] Extracting internal contents from .jar
[12:00:44 INFO]: [ItemsAdder] [License] Spigot product licensed to: Rampage_team (1724424)
[12:00:44 INFO]: [ItemsAdder] [Pack] DONE extracting internal contents from .jar
[12:01:02 INFO]: UUID of player BigTows_ is 37622b92-d064-35d3-92d7-64a1099b24d1
[12:01:04 ERROR]: [ItemsAdder] Cannot set model_id for item myitems:20-25lvl_staff Another item is already using model_id 12342 [myitems:dungeon_posoh]
[12:01:04 ERROR]: [ItemsAdder] If '[myitems:dungeon_posoh]' doesn't exists anymore you can run '/iacleancache items' command and then '/iareload'.
[12:01:04 ERROR]: [ItemsAdder] [STDERR] Model doesn't exists! /assets/custom_mobs/models/entity/walrus/.metadata
[12:01:04 WARN]: Nag author(s): '[LoneDev]' of 'ItemsAdder' about their usage of System.out/err.print. Please use your plugin's logger instead (JavaPlugin#getLogger).
[12:01:04 INFO]: Done (29.388s)! For help, type "help"
[12:01:04 WARN]: Can't keep up! Is the server overloaded? Running 20291ms or 405 ticks behind
[12:01:04 INFO]: Timings Reset
[12:01:04 INFO]: [ItemsAdder] Loaded 592 items
[12:01:04 INFO]: [ItemsAdder] Used 0/188 REAL block IDs
[12:01:04 INFO]: [ItemsAdder] Used 223/750 REAL_NOTE block IDs
[12:01:04 INFO]: [ItemsAdder] Used 1/63 REAL_TRANSPARENT block IDs
[12:01:04 INFO]: [ItemsAdder] Used 69/127 REAL_WIRE block IDs
[12:01:04 INFO]: [ItemsAdder] Used 5/14 FIRE block IDs
[12:01:04 INFO]: [ItemsAdder] Used 20/6608 font_images
[12:01:04 INFO]: [ItemsAdder] [Init] Loaded 16 categories
[12:01:04 INFO]: [ItemsAdder] [Init] Loaded successfully.
[12:01:05 INFO]: BigTows_ joined the game
[12:01:05 INFO]: BigTows_[/172.27.0.1:65143] logged in with entity id 1 at ([world]405.69971033201455, 141.0, 690.5800471501769)
[12:01:06 ERROR]: [ItemsAdder] [Pack] No URL set. Please follow the hosting tutorial http://a.devs.beer/first-respack-hosting
[12:01:17 INFO]: BigTows_ issued server command: //copy
[12:01:17 INFO]: Using com.sk89q.worldedit.bukkit.adapter.impl.fawe.v1_19_R1.PaperweightStarlightRelighterFactory as relighter factory.
[12:01:19 INFO]: BigTows_ issued server command: //paste
```

### Screenshots/Videos (you can drag and drop files or paste links)

![image](https://user-images.githubusercontent.com/9078504/227765185-ea8656a8-f4c0-46a7-836f-297a0f909db2.png)

**Comment by szymeqkox** [comment]
FAWE is required for other plugins to work, such as Arceon and GoBrush.

**Comment by LoneDev6** [comment]
Fixed in ItemsAdder-WorldEdit 1.1.3.
https://www.spigotmc.org/resources/addon-itemsadder-worldedit.79012/

**Comment by BigTows** [comment]
@LoneDev6 thx!
Works fine at 
1.19.4 - with all new versions of dependencies 
1.19.2 - with my build from issue.

Scenario was be successful:
```
//copy
//paste
//schematic save 
//schematic load 
//paste
```
